iceoryx_hoofs/include/iceoryx_hoofs/internal/relocatable_pointer/pointer_repository.hpp

```diff
@@ -109,6 +109,8 @@ class PointerRepository
 
     iox::cxx::vector<Info, CAPACITY> m_info;
     uint64_t m_maxRegistered{0U};
+
+    bool addPointerIfIndexIsFree(const id_t id, const ptr_t ptr, const uint64_t size) noexcept;
```

Comment:
```suggestion
    bool addPointerIfIdIsFree(const id_t id, const ptr_t ptr, const uint64_t size) noexcept;
```

Id and index into array are the same, on purpose to support fast lookup.

Reply:
Will take that to #1677